GoProjects/CalorieCounter/food/plate.go

```diff
@@ -1,27 +1,34 @@
 package food
 import "fmt"
 type Plate struct {
-	Items []Food
+	FoodItems []Food `json:"fooditems"`
```

Comment:
Json naming convention is camelCase (https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format)